packages/front-end/enterprise/components/Dashboards/DashboardEditor/DashboardBlock/TimeSeriesBlock.tsx

```diff
@@ -0,0 +1,70 @@
+import { TimeSeriesBlockInterface } from "back-end/src/enterprise/validators/dashboard-block";
+import { expandMetricGroups } from "shared/experiments";
+import ExperimentMetricTimeSeriesGraphWrapper from "@/components/Experiment/ExperimentMetricTimeSeriesGraphWrapper";
+import useOrgSettings from "@/hooks/useOrgSettings";
+import { useDefinitions } from "@/services/DefinitionsContext";
+import { getMetricResultGroup } from "@/components/Experiment/BreakDownResults";
+import { BlockProps } from ".";
+
+export default function TimeSeriesBlock({
```

Comment:
might be good to rename this to ExperimentTimeSeriesBlock (and also the underlying model's "time-series" type). There are many types of time series displays that could be included in dashboards, most of which are not tied to experiments (safe rollouts, plain metric analysis, plus new stuff...)